### ISSUE TITLE
v4 changed formatting conflicts with rubocop

### ISSUE THREAD

**Issue by mscrivo** [ruby-syntax-tree/syntax_tree#174]
Hey Kevin,

Seems that formatting has changed in 4.0, causing conflicts with rubocop rules.

For example:

4.0 wants to change this: 

```ruby
module Migration
  Sequel.migration do
    down do
      DB.run(
        "
        SOME SQL STMT
      "
      )
    end
  end
end
```

to this:

```ruby
module Migration
  Sequel.migration do
    down { DB.run("
        SOME SQL STMT
      ") }
  end
end
```

and rubocop is complaining:

```
C: [Correctable] Layout/FirstMethodArgumentLineBreak: Add a line break before the first argument of a multi-line method argument list.
      DB.run(" ...
```

another example with puts:

before:

```ruby
  puts(
    '
      some string.
    '.gsub(/^\s+/, '')
  )
```

after:
```ruby
  puts('
      some string.
    '.gsub(/^\s+/, ''))
````

**Comment by kddnewton** [comment]
Oof, that's not right. I'll check.

**Comment by kddnewton** [comment]
Released in v4.0.1

**Comment by mscrivo** [comment]
Thank you!
